samples/msal-node-samples/nextjs-rsc-ts/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "who-next",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "build": "next build",
+    "prod": "next start",
+    "start": "next dev",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "@azure/msal-node": "1.16",
+    "@emotion/react": "^11.11.0",
+    "@emotion/styled": "^11.11.0",
+    "@mui/icons-material": "^5.11.16",
+    "@mui/material": "^5.13.3",
+    "@remix-run/node": "^1.16.1",
+    "@types/node": "20.1.1",
+    "@types/react": "^18.2.8",
+    "@types/react-dom": "18.2.4",
+    "eslint": "8.40.0",
+    "eslint-config-next": "13.4.1",
+    "next": "^13.4.4",
+    "react": "18.2.0",
+    "react-dom": "18.2.0",
+    "redis": "^4.6.6",
+    "typescript": "^5.1.3"
+  },
+  "devDependencies": {
+    "eslint-config-prettier": "^8.8.0",
+    "eslint-plugin-prettier": "^4.2.1",
```

Comment:
It's considered a bad practice to use Linters for formatting code. I've linked some sources below that explain why, but the tl;dr is you should use linters for linting and formatters for formatting. 😃 

Sources:
- https://eslint.org/blog/2020/05/changes-to-rules-policies/#what-are-the-changes
- https://typescript-eslint.io/linting/troubleshooting/formatting/
- https://prettier.io/docs/en/comparison.html
- https://prettier.io/docs/en/integrating-with-linters.html
- https://www.youtube.com/watch?v=Cd-gBxzcsdA
- https://www.youtube.com/watch?v=sSJBeWPIipQ